docs/content/latest/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -15,15 +15,149 @@ isTocNested: true
+
+The following tables describe metrics available via the Yugabyte Platform UI.
+
+### YSQL Ops and Latency
+
+| **Graph**             | **Description**                                              | **Alert Guidance**                                           | **Example**                     |
+| :-------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------- |
+| Total YSQL Ops / Sec  | The count of DELETE, INSERT, SELECT, and UPDATE statements through the YSQL API. <br>This does not include index writes. | An alert should be issued if count drops significantly lower than your average count, as this might indicate an application connection failure. In addition, an alert should be issued if the count is much higher than your average count, as this could indicate a DDoS, security incident, and so on.<br>It is recommended to coordinate this with the application team because there could be legitimate reasons for dips and spikes. | ![img](/images/yp/metrics1.png) |
+| YSQL Op Latency (Avg) | Average time (in milliseconds) of DELETE, INSERT, SELECT, and UPDATE statements through the SQL API. | An alert should be issued when the overall latency is close to or higher than your application SLA.<br>Note that the overall latency metric is less helpful for troubleshooting specific queries. It is recommended that the application track query latency.<br>Also note that there could be reasons your traffic experiences spikes in latency. For example, ad-hoc queries such as count(*) are executed. | ![img](/images/yp/metrics2.png) |
+
+### YCQL Ops and Latency
+
+| **Graph**             | **Description**                                              | **Alert Guidance**                                           | **Example**                     |
+| :-------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------- |
+| Total YCQL Ops / Sec  | The count of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the YCQL API. | An alert should be issued if count drops significantly lower than your average count, as this could indicate an application connection failure. | ![img](/images/yp/metrics3.png) |
+| YCQL Op Latency (Avg) | The average time (in milliseconds) of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the YCQL API. | An alert should be issued when latency is close to or higher than your application SLA. | ![img](/images/yp/metrics4.png) |
+| YCQL Op Latency (P99) | The average time (in milliseconds) of the top 99% of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the CQL API. | Information pending                                          | ![img](/images/yp/metrics5.png) |
```

Comment:
Added the following to the "Alert Guidance" column:
_If this value is significantly higher than expected, then it might be a cause for concern and you might want to issue an alert.
You should check whether or not there are consistent spikes in latency._